supported_components.md

```diff
@@ -29,43 +32,33 @@
 - 04M062N
 - 04M065N
 - 04M066
-- 04M077N
 - 04M081
 - 04M082
-- Ubuntu 18.04 LTS 32-bit
-- Ubuntu 18.04 LTS 64-bit
 - Ubuntu 20.04 LTS 64-bit
-- CentOS 7.8 64-bit
-- CentOS 8.2 64-bit
-- Debian 10.5 32-bit
-- Debian 10.5 64-bit
-- Debian 10.6 32-bit
-- Debian 10.6 64-bit
-- Yocto - Sumo, Kernel 4.15.18 64-bit (02F026L + 02G025A only)
+- Ubuntu 22.04 LTS 64-bit
+- Debian 10 32-bit
+- Debian 10 64-bit
+- Debian 11 64-bit
+- RHEL 9.1 64-bit
 
 #### **Compilation - tested kernel versions:**
-- 3.16
-- 4.4
 - 4.9
```

Comment:
No requirement to support/test Linux kernel 4.9 (or is it used from any distro to support?).

Reply:
Ok. I think we had it in our build tests list but it is no longer a LTS releases. Removing it from list.